src/settings/MappingProfiles/detailsSections/edit/InvoiceDetailSection/InvoiceAdjustments.js

```diff
@@ -89,7 +89,7 @@ export const InvoiceAdjustments = ({
                 optionLabel="name"
                 wrapperLabel={`${TRANSLATION_ID_PREFIX}.wrapper.acceptedValues`}
                 wrapperSources={[{
-                  wrapperSourceLink: '/finance/funds',
+                  wrapperSourceLink: '/finance/funds?limit=1000&query=cql.allRecords=1 sortby name',
```

Comment:
Do we need a `limit` param here?

Reply:
looks like without with param we received only first 10 item from the list